widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java

```diff
@@ -187,6 +226,16 @@ public Builder backgroundColorRes(Integer backgroundColorRes) {
             return this;
         }
 
+        public Builder onHoldIcon(Integer onHoldIcon) {
```

Comment:
just a question, why Integer instead int?

Reply:
Runtime value could be null - if its null then we dont override the buildtime value

Reply:
clear